docs_src/pipelinerun.md

```diff
@@ -1,23 +1,25 @@
 # Pipeline Run
 
 ## Prerequisites: 
-Pipeline setup needs to be done first, pipeline setup documentation be found [HERE](./pipelinesetup.md)
+Pipeline setup needs to be done first, pipeline setup documentation be found [HERE](./pipelinesetup.md).
 
 ---
 ## Introduction to Pipeline run flow and expectation
-1. Pipeline can be using differnt media type as input choice:
+The pipeline run is meant to bring the service up and perform inferencing on the selected input media. The output of the pipeline run provides the inference results for each frame regarding the media source such as text, barcode, etc; as well as the FPS (frames per second). Pipeline run provides many options in media type, system process platform type, and additional optional parameters. These options give you the oppotunity to compare what system process platform is better for your need. There are few choices you have to make as inputs to the pipeline run script:
```

Comment:
maybe it would be better to put the next numbering (media choice, input choice...) under a sub header

```suggestion
The pipeline run is meant to bring the service up and perform inferencing on the selected input media. The output of the pipeline run provides the inference results for each frame regarding the media source such as text, barcode, etc; as well as the FPS (frames per second). Pipeline run provides many options in media type, system process platform type, and additional optional parameters. These options give you the oppotunity to compare what system process platform is better for your need. 

### Run Options
```